cfme/tests/infrastructure/test_datastore_analysis.py

```diff
@@ -4,15 +4,18 @@
 from cfme.fixtures import pytest_selenium as sel
 from cfme.infrastructure import datastore, host
 from cfme.web_ui import flash, tabstrip as tabs, toolbar as tb, Quadicon
-from utils import conf, testgen
+from utils import conf, testgen, version
 from utils.providers import setup_provider
 from utils.wait import wait_for
 import pytest
 
 DATASTORE_TYPES = ('vmfs', 'nfs', 'iscsi')
 
-# rhevm not supported
-PROVIDER_TYPES = ('virtualcenter',)
+# rhevm supported in 5.3+
+PROVIDER_TYPES = version.pick({
+    version.LOWEST: ('virtualcenter',),
+    '5.3': ('virtualcenter', 'rhevm')
+})
```

Comment:
I considered that before merging, then decided it was okay, merged, and now I'm realizing that it's not okay. Sorry 'bout that. :)